asset/css/markbind.css

```diff
@@ -302,3 +302,50 @@ panel[minimized]:not([expanded])::before {
 li.footnote-item:target {
     background-color: #eee;
 }
+
+/* "Copy" code block button */
+pre {
+    position: relative;
+}
+
+pre > .copy-btn-icon {
```

Comment:
IMO `.copy-btn-icon` is specific enough without requiring the `pre >` to be specified.

Reply:
Is there a reason why `copy-btn-icon` and `copy-btn` are split? They seem to only be applied together on the same element. `copy-btn-icon` also seems to be setting the styles for the button itself, and not the icon.